evalytics/server/handlers.py

```diff
@@ -6,6 +6,9 @@
 class WelcomeHandler(tornado.web.RequestHandler):
     path = r"/"
 
+    def initialize(self, reader):
+        self.__reader = reader
```

Comment:
I'd take the chance to replace these double underscored names (from `__reader` to `_reader`). I'm pretty sure that the double underscore thing is not designed for this particular. 

> `.__variable` is often incorrectly considered superprivate, while it's actual meaning is just to namemangle to prevent accidental access

> It should not be used to create a private method. It should be used to avoid your method being overridden by a subclass or accessed accidentally.

Python will rename that var as `_WelcomeHandler_reader` so actually you may still access it.

Reply:
talked offline, creating issue